src/vivarium/framework/components/parser.py

```diff
@@ -224,7 +227,7 @@ def prep_component(self, component_string: str) -> tuple[str, tuple]:
         return path, cleaned_args
 
     @staticmethod
-    def _clean_args(args: list, path: str) -> tuple:
+    def _clean_args(args: list[str], path: str) -> list[str]:
         """Transform component arguments into a tuple, validating that each argument
```

Comment:
nit: if you're making these lists instead of tuples you should update the docstring.